extensions/amp-story/1.0/amp-story-system-layer.css

```diff
@@ -235,14 +234,39 @@
   display: none !important;
 }
 
+/*
+  Visible to screen readers only.
+  https://www.a11yproject.com/posts/how-to-hide-content/
+*/
+.i-amphtml-paused-display {
+  position: absolute !important;
```

Comment:
We just need `position: absolute; opacity: 0;` or sth as simple as that, I think

Reply:
+1, with a `pointer-events: none;` so it can't be clicked

Reply:
@mszylkowski came up with an alternative selector to use, switched to that and simplified the rules.